### PR TITLE
Remove version from windows dev release url

### PR DESCRIPTION
**Pull request by wojtekn**
## Proposed Changes

I propose to remove version from the Windows dev build URL as only commit is used for dev builds.

## Testing Instructions

Merge and confirm deb build URLs are correct in the manifest file.

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: scripts/generate-releases-manifest.mjs [hunk 1]
```diff
@@ -151,7 +151,7 @@ if ( isDevBuild ) {
 	const windowsReleaseInfo = await getWindowsReleaseInfo();
 	releasesData[ 'dev' ][ 'win32' ] = {
 		sha: windowsReleaseInfo.sha1,
-		url: `${ cdnURL }/${ baseName }-win32-v${ version }-${ currentCommit }-full.nupkg`,
+		url: `${ cdnURL }/${ baseName }-win32-${ currentCommit }-full.nupkg`,
```
Comment (fluiddot) [inline scripts/generate-releases-manifest.mjs:154]: @wojtekn The version is needed for parsing the release information by Squirrel. If we expect dev builds to be updated, we'd need to keep this part.

Comment (wojtekn) [reply scripts/generate-releases-manifest.mjs:154]: @fluiddot I wasn't sure which was correct - adding the version in both places or removing it - so I removed it to at least point it to the existing file. If we need to add a version, we must first change the `fastlane/Fastfile` to use the version and hash for the dev build file name.

Comment (fluiddot) [reply scripts/generate-releases-manifest.mjs:154]: > If we need to add a version, we must first change the fastlane/Fastfile to use the version and hash for the dev build file name.

Ah, good point. Yes, we'd need to incorporate the current version (extracted from `package.json`) into the filename.

Comment (wojtekn) [reply scripts/generate-releases-manifest.mjs:154]: Thanks, added https://github.com/Automattic/dotcom-forge/issues/7439

**Pull request merged**